### DISCUSSION TITLE
Tag Creator limit length tags

### DISCUSSION THREAD

**Discussion by ninhorata** [formkit/formkit#1019] [General]
Hello!

I'm using the TagCreator component, but I don't see or cannot find a way to limit the max number of characters per tag. So let's say tags can be length:3,20 but it doesn't work. Has somebody done this before?

Thanks!

**Comment by sashamilenkovic** [comment]
@ninhorata #1092 This issue is similar. Here's' an example of how to write your own `allow-new-values` behavior by using a FormKit plugin and higher order functions. https://formkit.link/bc2cd1df26e93c21f845615828292e7e

Hope this is helpful, let me know if you have any questions.
